src/main/java/com/facebook/openwifirrm/optimizers/channel/RandomChannelInitializer.java

```diff
@@ -46,6 +50,27 @@ public RandomChannelInitializer(
 		super(model, zone, deviceDataManager);
 	}
 
+	/** Constructor (allows setting different channel per AP) */
+	public RandomChannelInitializer(
+		DataModel model, String zone, DeviceDataManager deviceDataManager, boolean setDifferentChannelsPerAp
+	) {
+		this(model, zone, deviceDataManager);
+		this.setDifferentChannelsPerAp = setDifferentChannelsPerAp;
+	}
+
+	/** Constructor (allows setting different channel per AP and passing
+	 * in a custom Random class to allow seeding) */
```

Comment:
(since we've disabled comment autoformatting...)
If a Javadoc comment does not fit in one line, keep the `/**` and `*/` on their own separate lines.
```java
/**
 * Long comment
 */
```